Package.swift

```diff
@@ -3,21 +3,23 @@
 import PackageDescription
 import Foundation
 
-var swiftSyntaxUnsafeSwiftFlags: [String] = []
-
 /// If we are in a controlled CI environment, we can use internal compiler flags
 /// to speed up the build or improve it.
-if ProcessInfo.processInfo.environment["SWIFT_BUILD_SCRIPT_ENVIRONMENT"] != nil {
+let swiftSyntaxSwiftSettings:[SwiftSetting] 
```

Comment:
Nitpick to match the existing style in SwiftSyntax
```suggestion
let swiftSyntaxSwiftSettings: [SwiftSetting] 
```